### PR TITLE
S8#issue

### PR DESCRIPTION
**Pull request by DevPH-Wistkey**

### CODE REVIEW

File: components/SSI/AddFunds/index.tsx [hunk 1]
```diff
@@ -529,15 +529,32 @@ function Component(props: InputType) {
...
                   {originator_address.username === undefined && (
                     <p style={{ marginBottom: "10%" }}>
                       About to send funds from{" "}
-                      {zcrypto.toBech32Address(originator_address?.value)}
+                      {zcrypto.toBech32Address(originator_address?.value)}{" "}
+                      into&nbsp;
+                      <span style={{ color: "#ffff32" }}>
+                        {loginInfo?.username
+                          ? `${loginInfo?.username}.did`
+                          : zcrypto.toBech32Address(loginInfo?.address)}{" "}
```
Comment (tralkan) [inline components/SSI/AddFunds/index.tsx:556]: this situation should never happen atm

File: components/SSI/AddFunds/index.tsx [hunk 2]
```diff
@@ -683,20 +719,6 @@ function Component(props: InputType) {
                 )} */}
                 {
                   <>
-                    <h3 style={{ marginTop: "7%" }}>
-                      Add funds into{" "}
-                      {type === "buy" ? (
```
Comment (tralkan) [inline components/SSI/AddFunds/index.tsx:688]: not sure this should be removed @DevPH-Wistkey

**Review by tralkan — CHANGES_REQUESTED** [review]
let's add the [latest card](https://www.notion.so/ssiprotocol/Transaction-Status-Minimized-f4187d614587432498790e8bf19fd7a2) into this PR @DevPH-Wistkey

File: components/SSI/AddFunds/index.tsx (hunk 2 above)
Comment (DevPH-Wistkey) [reply components/SSI/AddFunds/index.tsx:688]: Hi @tralkan I'm referring to Sherly comment here: https://www.notion.so/ssiprotocol/s8-issue-Sherly-52dc2afd7bdd43159a68f20dee8ed888#d3abb4da64e4427cbe0757cad1ff3ce9

File: components/SSI/AddFunds/index.tsx (hunk 1 above)
Comment (DevPH-Wistkey) [reply components/SSI/AddFunds/index.tsx:556]: Then, you want me to put only `zcrypto.toBech32Address(loginInfo?.address)` ?

File: components/SSI/AddFunds/index.tsx (hunk 2 above)
Comment (tralkan) [reply components/SSI/AddFunds/index.tsx:688]: I see but I'll find another way to summarise.

File: components/SSI/AddFunds/index.tsx (hunk 1 above)
Comment (tralkan) [reply components/SSI/AddFunds/index.tsx:556]: I mean that there's always gonna be a username logged in when adding funds from /balance. You can leave it as it is for now.

Comment (DevPH-Wistkey) [reply components/SSI/AddFunds/index.tsx:556]: Noted

File: components/SSI/AddFunds/index.tsx (hunk 2 above)
Comment (DevPH-Wistkey) [reply components/SSI/AddFunds/index.tsx:688]: Should I put back the deleted component? @tralkan

File: components/Modals/BuyNFT/index.tsx [hunk 3]
```diff
@@ -381,233 +400,243 @@ function Component() {
...
-                    display: "flex",
-                    width: "100%",
-                    justifyContent: "center",
-                  }}
-                >
-                  <p style={{ marginTop: "1%" }}>To continue:&nbsp;</p>
-                  <button className="button" onClick={handleConnect}>
-                    <p>LOG IN</p>
-                  </button>
+            ) : txType === "AddFunds" ? (
```
Comment (tralkan) [inline components/Modals/BuyNFT/index.tsx:419]: there is a mistake here/ not needed @DevPH-Wistkey

**Review by tralkan — CHANGES_REQUESTED** [review]

File: components/Modals/BuyNFT/index.tsx (hunk 3 above)
Comment (DevPH-Wistkey) [reply components/Modals/BuyNFT/index.tsx]: Updated, please kindly check it @tralkan

File: components/SSI/AddFunds/index.tsx (hunk 2 above)
Comment (tralkan) [reply components/SSI/AddFunds/index.tsx:688]: yes please @DevPH-Wistkey

Comment (DevPH-Wistkey) [reply components/SSI/AddFunds/index.tsx]: Solved

**Pull request merged**